docs/modules/operation/pages/quick-start/performance-data.adoc

```diff
@@ -0,0 +1,50 @@
+
+= Step 4: Set Up Performance Data Collection
+
+This section describes how to set up performance data collection in {page-component-title}.
+
+{page-component-title} collects performance data using xref:reference:daemons/daemon-config-files/collectd.adoc[collectd], which is enabled by default.
+Collectd schedules data collection on network entities using management agents and protocol-specific collectors.
+
+NOTE: In the {page-component-title} docs, any reference to collectd refers to the {page-component-title} collect daemon, not the open source project of the same name.
+
+Data collection for SNMP and OpenNMS-JVM is enabled by default, allowing {page-component-title} to monitor itself through JMX.
+
+For additional information, including collector configuration, see xref:operation:deep-dive/performance-data-collection/introduction.adoc[Performance Management] in the Deep Dive section.
+
+== Before you begin
+
+{page-component-title} data collection for SNMP works out of the box, provided your SNMP credentials are configured correctly.
+
+The default value for the community string is `public`.
+
+NOTE: If you are using SNMPv3, you need to configure details for a User-Based Security Model (USM) user instead of a community string.
+{page-component-title} supports SNMPv3 and USM, but these topics are outside of the scope of this guide.
+
+If your community string is different, you need to change it by following these steps:
+
+. In the web UI, click the *Gear* symbol at the top-right of the screen.
+. Choose menu:Provisioning[Configure SNMP Community Names by IP Address].
+. In the v1/v2c Specific Parameters section, enter the new community string in the *Read Community String* box.
+. Click *Save Config* to save your changes.
+
+Performance data collection on other protocols (HTTPS, JMX, and so on) requires additional configuration.
+You may also want to change how collectd works—when, how, and what data it collects.
+For more information, see xref:reference:performance-data-collection/introduction.adoc[Collectors].
+
+== Default performance data collection
+
+By default, {page-component-title} collects a large variety of information from monitored nodes that support SNMP.
+This information includes widely-implemented metrics that are standard for MIB-2 sources (for example, TCP group data, network interface counters).
+SNMP data collection definitions are also included for devices from a range of popular vendors (for example, if you add a Cisco router to your instance, {page-component-title} will automatically collect Cisco-specific metrics).
+The metrics included in these collections vary by vendor.
```

Comment:
I'm a bit confused due to collections, definitions, metrics, and what distinctions we need to make.

Do the definitions define the metric collected? Does "collections" here refer to the data that is collected or the definitions?

Reply:
"Definitions" is a blanket term for "what's in the relevant config files".
"Collections" refers to the container which pulls together the "what" and the "where" of what gets collected. This idea maps to the `snmp-collection` tag in `etc/datacollection-config.xml`.
"Metrics" are the "what" inside the collection. This idea maps to the `mibObj` tag in the same file.

I can go deeper if it will be helpful, but I'm cutting myself off here unless you request more.